mypy/main.py

```diff
@@ -174,6 +174,9 @@ def process_options(args: List[str],
     parser.add_argument('--strict-optional', action='store_true',
                         dest='special-opts:strict_optional',
                         help="enable experimental strict Optional checks")
+    parser.add_argument('--strict-optional-whitelist', metavar='GLOB', nargs='*',
```

Comment:
Can you manually also turn on `--strict-optional` if this is not None, so I won't have to write `--strict-optional --strict-optional-whitelist`?